icubam/www/handlers/update.py

```diff
@@ -72,7 +72,7 @@ def get_icu_data_by_id(self, icu_id, def_val=0):
 
     except Exception as e:
       logging.error(e)
-      return self.redirect('/error')
+      return self.set_status(400)
```

Comment:
Wouldn't it be better to return 404 ? Be sure that returning 400 is not a way to tell hackers "try again".